androidApp/src/main/java/dev/johnoreilly/confetti/di/AppModule.kt

```diff
@@ -33,9 +35,11 @@ val appModule = module {
     viewModelOf(::SpeakersViewModel)
     viewModelOf(::SessionDetailsViewModel)
     viewModelOf(::SpeakerDetailsViewModel)
-    viewModelOf(::AccountViewModel)
     viewModelOf(::SettingsViewModel)
     viewModelOf(::SearchViewModel)
+    viewModelOf(::SessionAndroidViewModel)
+    viewModelOf(::SpeakerAndroidViewModel)
+    viewModelOf(::SearchAndroidViewModel)
```

Comment:
This works runtime, but it gives me an red line in the IDE since it does not directly extend the KMMViewModel abstract class

Reply:
I got this too. I haven't raised a bug, but unless there is a simple fix, maybe we should.